Sources/XCTest/Public/XCTestMain.swift

```diff
@@ -87,6 +87,33 @@ public func XCTMain(_ testCases: [XCTestCaseEntry]) -> Never {
     case .list(type: .json):
         TestListing(testSuite: rootTestSuite).printTestJSON()
         exit(0)
+    case let .help(invalidOption):
+        if let invalid = invalidOption {
+            print("Error: Invalid option \"\(invalid)\"\n")
```

Comment:
Could this `print` be output to stderr instead of stdout?

Reply:
Makes sense to me... but is there an easier way to print to stderr than what I could figure out?